brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1334 Fix: Mareting Button - title hover color not applying to icon in the editor

### Description
Marketing button: Heading color is applying for icon also but hover color is not applying for icon

### Screenshots
Issue - https://d.pr/v/YAmuaK
Fix - https://share.getcloudapp.com/llu0g0B7

### Types of changes




Bug Fix

### How has this been tested?
- Add a marketing button 
- Go to style setting
- add heading color and hover color
- check on frontend and backend

### Checklist:
